### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by Matthensley777**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+    var CarLot = (function() {
+
+
+        function populatePage(inventory) {
+            // Loop over the inventory and populate the page
+
+
+document.getElementById("text").addEventListener("keyup", carBoundToText);
+
+function carBoundToText(e) {
```
Comment (morecallan) [inline main.js]: Is this function doing anything in this file? It looks like you might be doing it in your print.js instead.

File: print.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    function selectCar(e) {
+        var cars = document.getElementsByClassName("container");
+        console.log(container)
+        for (var j = 0; j < cars.length; j++) {
+            cars[j];
+        }
+
+        if (e.target.classList.contains("container")) {
+            e.target.style.border = "solid black 5px";
```
Comment (morecallan) [inline print.js]: Avoid inline styling like this. Try putting these in a CSS class and adding the class to the target instead.

File: print.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    var currentCar;
+
+    //  events if/else statements. 
+    carContainer.addEventListener("click", selectCar);
+
+    function selectCar(e) {
+        var cars = document.getElementsByClassName("container");
+        console.log(container)
+        for (var j = 0; j < cars.length; j++) {
+            cars[j];
```
Comment (morecallan) [inline print.js]: What's the plan for this for-loop?

**Review by morecallan — COMMENTED** [review]
@Matthensley777 - Looks like you are making progress on this. I'm not able to run the project yet - looks like you have a few things you are in the middle of working on but I wanted to give you some direction so I wrote inline comments. 

Also, make sure you are sticking with the iife/augmenter structure. (When you want functions to be used by other files, you have to "export" them in the return). You are doing this in your main IIFE but the other ones are just returning the old one and not exporting the functions you are writing, like in print.js, the selectCar function isn't part of your CarLot IIFE. Take a look at old projects to see how those iife augmenters work. 

Looks like you are working towards a solution on this. We are now signed back into Slack if you have questions.

**Comment by zoeames** [comment]
@Matthensley777 i am still seeing a bunch of bugs - card color doesn't change, description changes as i type but then reverts back to the old bio.  sit with callan or lauren on monday

**Pull request merged**